posthog/models.py

```diff
@@ -222,8 +224,12 @@ def create(self, *args: Any, **kwargs: Any):
             for action in event.actions:
                 relations.append(action.events.through(action_id=action.pk, event=event))
             Action.events.through.objects.bulk_create(relations, ignore_conflicts=True)
-            return event
+            for action in event.actions:
+                if action.post_to_slack and action.team and action.team.slack_incoming_webhook:
+                    values = {'text': 'Action "{}" triggered!'.format(action.name)}
+                    requests.post(action.team.slack_incoming_webhook, verify=False, json=values)
```

Comment:
With the latest rebase, I had to make some funky changes here (two loops)
However this is a WIP anyway, as we'll move this to workers soon enough.

Reply:
Would you mind moving this up into the property above so it is one loop? The `event.actions` is a really nasty SQL query and in this case we're running it twice.